nornir/plugins/functions/text/__init__.py

```diff
@@ -77,14 +89,25 @@ def _print_result(
     attrs: List[str] = None,
     failed: bool = False,
     severity_level: int = logging.INFO,
-) -> None:
+    return_output: bool = False,
```

Comment:
same as below, instead of using a bool just pass here the destination where you want to write the `print`. Difference here is that the parameter should be mandatory.